packages/apptive_grid_form/lib/src/widgets/form_widget/attachment_form_widget.dart

```diff
@@ -14,11 +14,15 @@ class AttachmentFormWidget extends StatefulWidget {
   const AttachmentFormWidget({
     super.key,
     required this.component,
+    this.enabled = true,
   });
 
   /// Component this Widget should reflect
   final FormComponent<AttachmentDataEntity> component;
 
+  /// Flag whether the widget is enabled
+  final bool enabled;
```

Comment:
I really don't like how this needs to be added to each widget which means a lot of potential missing/inconsistent ways of dealing with this. I'm wondering if it might make sense to introduce an inherited widget that handles this at one place and all the form widget are then wrapped by one widget and `enabled`/`disabled` are checked with a lookup to that inherited widget.
What do you think? / Have you considered this option?

Reply:
Looking through some more stuff wouldn't it make even more sense if the whole properties would be stored in the FormComponent rather than just the form data? Did you explore these options?

Reply:
Yeah, I like the option of parsing it into the FormComponents, I changed it to that.